payments/card-payments/direct.md

```diff
@@ -0,0 +1,355 @@
+customers that are compliant with PCI-DSS regulations**, and is a way to
+implement card payments without using Swedbank Pay Hosted payment pages." %}
+
+### Purchase flow
+
+* The payer places an order and you make a `POST` request towards Swedbank Pay
+  with gathered `Purchase` information.
+* The action taken next is the `direct-authorization` operation that is returned
+  in the first request. You `POST` the payer's card data to the URL in
+  the[`direct-authorization` operation][authorization].
+* If 3-D Secure authentication is required, you will then receive a URL where
+  you will have to redirect the payer.
+* When the payment is completed, the payer needs to be redirected back to your
+  merchant/webshop site.
+* Finally you make a `GET` request towards Swedbank Pay with the `paymentID`
+  received in the first step, which will return the purchase result.
+
+The sequence diagram below shows a high level description of a complete
+purchase, and the requests you have to send to Swedbank Pay. The links will take
+you directly to the corresponding API description.
+
+```mermaid
+sequenceDiagram
+    participant Payer
+    participant Merchant
+    participant SwedbankPay as Swedbank Pay
+
+    activate Payer
+    Payer->>+Merchant: start purchase
+    deactivate Payer
+    Merchant->>+SwedbankPay: POST /psp/creditcard/payments
+    deactivate Merchant
+    note left of Merchant: First API Request
+    SwedbankPay-->>+Merchant: rel: direct-authorization
+    deactivate SwedbankPay
+    Merchant->>+SwedbankPay: GET <payment.id>
+    deactivate Merchant
+    note left of Merchant: Second API request
+    SwedbankPay-->>+Merchant: rel: direct-authorization
+    deactivate SwedbankPay
+    Merchant-->>-Payer: display purchase result
+```
+
+### API Requests
+
+The API requests are displayed in the [purchase flow](#purchase-flow-mobile).
+You can [create a card `payment`][create-payment] with following `operation`
+options:
+
+* [Purchase][purchase]
+* [Recur][recur]
+* [Payout][payout]
+* [Verify][verify]
+
+Our `payment` example above uses the [`purchase`][purchase] value which is
+explained below.
+
+### Purchase
+
+A `Purchase` payment is a straightforward way to charge the card of the payer.
+It is followed up by posting a capture, cancellation or reversal transaction.
+
+An example of an abbreviated `POST` request is provided below. Each individual
+Property of the JSON document is described in the following section.
+An example of an expanded `POST` request is available in the
+[other features section][purchase].
+
+{% include alert.html type="neutral" icon="info" body="
+Please note that in order to minimize the risk for a challenge request
+(Strong Customer Authentication – “SCA”) on card payments, it's recommended that
+you add as much data as possible to the `riskIndicator` object in the request
+below." %}
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/creditcard/payments HTTP/1.1
```

Comment:
This code example is probably reused so it can be extracted into an include, no?

Reply:
will  fix this after merge